identity/registry/transactor.go

```diff
@@ -229,6 +229,33 @@ func (t *Transactor) FetchStakeDecreaseFee(chainID int64) (FeesResponse, error)
 	return f, err
 }
 
+// CombinedFeesResponse represents the combined fees response.
+type CombinedFeesResponse struct {
+	Current    Fees      `json:"current"`
+	Last       Fees      `json:"last"`
+	ServerTime time.Time `json:"server_time"`
+}
+
+// Fees represents fees for a given time frame.
+type Fees struct {
+	DecreaseStake *big.Int  `json:"decreaseStake"`
+	Settle        *big.Int  `json:"settle"`
+	Register      *big.Int  `json:"register"`
+	ValidUntil    time.Time `json:"valid_until"`
+}
+
+// FetchCombinedFees fetches current transactor fees.
+func (t *Transactor) FetchCombinedFees(chainID int64) (CombinedFeesResponse, error) {
+	f := CombinedFeesResponse{}
+	req, err := requests.NewGetRequest(t.endpointAddress, fmt.Sprintf("fee/%v", chainID), nil)
```

Comment:
maybe use the cache, just to make sure this endpoint is not overused in node ui and results in a lot of calls to transactor?

Reply:
I dont like the cache with time sensitive expiration for the fees. The cache also doesnt fit this struct so I would have to re-write how it works. a bit annoying.

Reply:
then we can just remove it for all I guess